samples/ConsoleApp/Program.cs

```diff
@@ -11,6 +11,9 @@ internal class Program
 
         private static async Task<int> Main()
         {
+            Console.WriteLine("Ensure right version of DiagnosticSource is installed");
+            Console.WriteLine(typeof(ActivitySource).Assembly.FullName);
```

Comment:
I will test this myself later but I'm assuming that with our always inject option the SDK was already injected by this point. Anyway I must confirm that.